Applications/Spire/Source/KeyBindings/OrderTasksPage.cpp

```diff
@@ -141,14 +143,15 @@ namespace {
 
   auto display(const Region& region) {
     auto text = QString();
+    auto delimiter = QString(0x1f);
```

Comment:
Why is a delimiter used to display text?

Reply:
The purpose of creating the function is to generate a string of the region for searching. It can be useful to use delimiters in searching.
For example, suppose a region includes "USA" and "AUS", if no delimiter is used, the string of the region might be created as "USAAUS". The user types "AA" in the search box, the region will meet the filter, but actually it shouldn't.

Reply:
A function named ```display``` shall only be used to display a text representation of a value. If you're intention is to perform a search then you will need a function that serves that specific purpose.